### ISSUE TITLE
Remove email addresses from public pages 

### ISSUE THREAD

**Issue by krrisis** [gitcoinco/web#7624]
### Who
Could you please remove mail addresses from bounties? You can see my personal mail address if you scroll eg to the very bottom of [this github bounty](https://gitcoin.co/issue/aragon/aragon-monthly/284/1850)
I assume they're also still visible by google & other scrapers in other inactive bounties but can't seem to browse to these. 

Mentioned this before in the townsquare (where someone responded) bcs I didn't want to get scraped. But can't find my own request with search :/ , so logging it here again. Just was reminded of it bcs I found myself in a pastebin file via yougotpwnd, leading to... gitcoin. :/ 

thx & keep up the good work

**Comment by krrisis** [comment]
FYI, just as a reference (as I still had it open :) ), [this is the pastebin](https://pastebin.com/WdxfKBEL) that's been leaked with my mail address in it that I found. This scrape alone has about 200+ mail addresses in it, all seem to come from gitcoin a.o. @vs77bb's gmail address & @owocki's gitcoin address are in here. Will prob edit this comment later - or you can do so, to not get even more google crawlers leading to your personal mail inboxes ;-)

**Comment by owocki** [comment]
Kris; this is why we have a "It's okay to show this information publicly on Gitcoin" toggle checkbox upon bounty creation.

https://bits.owocki.com/Blu1NoNd

**Comment by krrisis** [comment]
@owocki Yeah, I vaguely remember that (been a while) and I assumed that only referred to making the funding organization & github user name public, didn't think people ever want their mail addresses publicly available and crawl-able on sites... 

In any case, is there any way for me to take this down?

Fun fact, the above link to your site triggered bitdefender on windows & blocked the link for phishing, had to check it on my phone :)
